VirtoCommerce.Platform.Web/Startup.cs

```diff
@@ -529,6 +541,62 @@ private static void InitializePlatform(IAppBuilder app, IUnityContainer containe
                         }
                     },
                     new ModuleSettingsGroup
+                    {
```

Comment:
We don't need a create platform settings that can be changed in run-time, let's restricting only settings that being  reading from web.config

Reply:
Done